### PR TITLE
Use a simpler behavior on volume bar , closes #351

### LINKED ISSUE #351: Volume bar opens to left of volume icon and covers the play buttons
**The volume bar opens at the left of the volume icon and covers the play buttons** until you move the mouse away. Sometimes you want to click pause you move your mouse over the volume icon and you need to make a second move to close the volume bar that pops up. Being on the right side of volume icon makes it more simple.

screenshot from 2017-07-23 13-20-02
screenshot from 2017-07-23 13-21-43

My suggestion is to make it open at the right of the icon 😉 

**Specs**: 
Museeks 0.8.1 (Installed via AUR) -- OS: _Arch Linux_ -- Kernel: _x86_64 Linux 4.11.7-1-ARCH_

### PR DESCRIPTION
**Pull request by vprigent**
Closes #351

Drop volume bar show/hide behavior
Update style for volume bar

I took some considerations on the new position :
I didn't wanted it to be too close of the player infos, as it is still a control element.
And still, wanted to separate it a little from the play/next/previous controls.

### CODE REVIEW

**Comment by KeitIG** [comment]
Can you post a screenshots please ? ^^

**Comment by KeitIG** [comment]
And check eslint too ;)

File: src/styles/_header.scss [hunk 1]
```diff
@@ -396,9 +396,9 @@ header {
         }
 
         &.volume {
-            padding: 0;
-            width: 34px;
             position: relative;
+            display: inline-flex;
+            padding-left: 1em
```
Comment (YurySolovyov) [inline src/styles/_header.scss]: semicolon

File: src/styles/_header.scss [hunk 2]
```diff
@@ -408,23 +408,19 @@ header {
...
+        height: 20px;
 
         input[type=range] {
-            width: 88px;
-            height: 20px;
             background-color: transparent;
         }
+
+        + span {
+          margin: 2px 6px 0
```
Comment (YurySolovyov) [inline src/styles/_header.scss]: semicolon

**Comment by vprigent** [comment]
My bad, i'll do better next time !

CI should pass now

screenshot-museeks

**Comment by KeitIG** [comment]
Sorry for the delay, I will try to test it today, I might ask for a few changes

**Comment by YurySolovyov** [comment]
@vprigent you might need to rebase, some commits landed on master

**Comment by KeitIG** [comment]
Two things:

- I still need to test it
- I want to know why some CI tests are not triggered

**Comment by YurySolovyov** [comment]
My guess is that they may be skipped because of merge conflicts, so they don't even try

**Comment by KeitIG** [comment]
I think it's because it is a PR from a Fork

**Comment by KeitIG** [comment]
I think I fixed it for CircleCI, but not for AppVeyor

**Comment by KeitIG** [comment]
Ok, a few things, why did you drop the show/hide feature?

->

screenshot from 2017-09-08 13 36 06

Here's the problem. 

Initially I based the volume design on deezer.com player, the problem being: how to place a volume controller when you don't have enough place for it.

I am closing it, let's discuss about problems and possible solutions in #352

**Comment by YurySolovyov** [comment]
You meant #351 probably

**Pull request closed without merging**